docs/commands/graph.slowlog.md

```diff
@@ -18,3 +18,11 @@ GRAPH.SLOWLOG graph_id
     3) "MATCH (me:Person)-[:FRIEND]->(:Person)-[:FRIEND]->(fof:Person) RETURN fof.name"
     4) "0.288"
 ```
+
+To reset a graph's slowlog issue the following command:
+
+```sh
+GRAPH.SLOWLOG graph_id RESET
```

Comment:
didn't we prefer 
GRAPH.SLOWLOG RESET graph_id
so it be subcommand?
can acl work like this?

Reply:
I didn't want to create ambiguity with `GRAPH.SLOWLOG <key>`
the key is always at position `argv[1]`

Reply:
@MeirShpilraien help?

Reply:
I talked with @MeirShpilraien there are 2 options
1. introduce new command GRAPH.SLOWLOG2 that have 2 subcommands GRAPH.SLOWLOG2 GET ket GRAPH.SLOWLOG2 RESET key and deprecate GRAPH.SLOWLOG
2. introduce new command GRAPH.RESET_SLOWLOG key

Reply:
from a talk with @gkorland and @LiorKogan we can leave it like this